Samples/WorkflowCombineSampleApp/README.md

```diff
@@ -0,0 +1,48 @@
+# WorkflowCombineSampleApp 
+
+This sample project utilizes the WorkflowCombine library to demonstrate its usage in a `Workflow`. It is a simple app with a label that updates the current date & time every second.
+
+# Usage
+
+Thanks to the `AnyWorkflowConvertible` protocol, both the `WorkflowReactiveSwift` and `WorkflowCombine` Workers have identical api interface. To to migrate your `WorkflowReactiveSwift` `Worker`s to use the `WorkflowCombine`  `Worker`s, you will need to do the following:
```

Comment:
Let's drop `ReactiveSwift` mentions from here.

Reply:
Removing references of `ReactiveSwift` kinda makes this readme not as useful. Would you rather me move it to a different document somewhere?